src/main/resources/lang/default.lang

```diff
@@ -2275,6 +2275,18 @@ transform reasons:
 	unknown: unknown
 	infection: infection, villager infection
 
+# -- Item Flags --
+item flags:
+	hide_additional_tooltip: hide additional tooltip
+	hide_armor_trim: hide armor trim
+	hide_attributes: hide attributes
+	hide_destroys: hide destroys
+	hide_dye: hide dye
+	hide_enchants: hide enchants
+	hide_placed_on: hide placed on
+	hide_stored_enchants: hide stored enchants
+	hide_unbreakable: hide unbreakable
```

Comment:
i'd like to see some more grammatical options as well
Like "hide enchantments" and "hide destroyable blocks" or things like that.

Reply:
Could some valid options also be the same things but without `hide` in the name? Like for the `hide_enchants` one, could the options be `hide enchants, enchants, hidden enchants` (with `enchantments` as options too)?